tests/5.0/requires/test_requires_unified_shared_memory_static_is_device_ptr.F90

```diff
@@ -0,0 +1,84 @@
+!===--- test_requires_unified_shared_memory_static_is_device_ptr.F90 --------------===//
+!
+! OpenMP API Version 5.0 Nov 2018
+!
+! We request the use of unified_shared_memory in this program.
+! Checking for static arrays. The array is global and then accessed through 
+! a pointer from the host and the device.
+!
+! We use is_device_ptr to guarantee that the host pointer is used in the device.
+!
+!===------------------------------------------------------------------------------===//
+
+#define OMPVV_MODULE_REQUIRES_LINE !$omp requires unified_shared_memory
+#include "ompvv.F90"
+
+#define N 1024
+
+PROGRAM test_requires_unified_shared_memory_static_is_device_ptr
+   USE iso_fortran_env
+   USE ompvv_lib
+   USE omp_lib
+   implicit none
+   LOGICAL:: isOffloading
+   ! STATIC ARRAY
+   INTEGER, TARGET, DIMENSION(N):: anArray
+
+!$omp requires unified_shared_memory
+
+  OMPVV_TEST_AND_SET_OFFLOADING(isOffloading)
+
+  OMPVV_WARNING_IF(.NOT. isOffloading, "With no offloading, unified shared memory is guaranteed due to host execution")
+
+  OMPVV_TEST_VERBOSE(unified_shared_memory_static_is_device_ptr() .NE. 0)
+
+  OMPVV_REPORT_AND_RETURN()
+
+CONTAINS
+  INTEGER FUNCTION unified_shared_memory_static_is_device_ptr()
+    INTEGER:: errors, i
+    INTEGER, DIMENSION(N):: anArrayCopy
+    INTEGER, POINTER:: aPtr(:)
+
+    OMPVV_INFOMSG("Unified shared memory testing - Static Array")
+
+    errors = 0
+    aPtr => anArray
+
+    DO i = 1, N
+      anArray(i) = i
+      anArrayCopy(i) = 0
+    END DO
+
+    ! Test for writes to this varriable from device
+    !$omp target is_device_ptr(aPtr) 
```

Comment:
Same comment as in #585 applies here:  This violates the 5.0 restrictions for **is_device_ptr**, but it is valid in 5.1. Thus, either it should be moved to directory 5.1 – or the argument must be changed into a dummy argument without POINTER attribute.

Reply:
Updated the Fortran version to use dummy argument in the is_device_ptr clause.